data/src/test/java/org/oppia/android/data/backends/gae/api/ClassroomServiceTest.kt

```diff
@@ -27,7 +27,7 @@ class ClassroomServiceTest {
   @Before
   fun setUp() {
     val client = OkHttpClient.Builder()
-    client.addInterceptor(NetworkInterceptor())
+    client.addInterceptor(JsonPrefixNetworkInterceptor())
```

Comment:
Please also inject this everywhere it's used in tests, too.

Though, it might be helpful to wait for #2642 if we can since I think that's largely simplifying data test setup.

Reply:
Is this interceptor needed for the mock retrofit? All the service tests pass when removing it, maybe that's an indicator that our mock retrofit is not set up properly.

I ask since injecting the interceptors means we then need to declare TestApplications for all the tests that use the interceptors, is that right?

Reply:
Based on our discussion offline I'm going to leave the interceptor here so that it doesn't interfere with #2642 since it seems that the interceptors will be added there.

Reply:
Moved this comment to #2916 via https://github.com/oppia/oppia-android/pull/2916/files#r610890818.